### PR TITLE
feat: add linkedin extension support

### PR DESCRIPTION
**Pull request by sambhavgupta0705**
this pr add essential schemas to add support for linkedin extension
https://github.com/asyncapi/extensions-catalog/issues/78
cc: @derberg

### CODE REVIEW

File: extensions/linkedin/0.1.0/schema.json
Comment (derberg) [inline extensions/linkedin/0.1.0/schema.json:2]: how about we also add `pattern` field, like `^https:\/\/(?:www\.)?linkedin\.com.*` that will validate provided link to make sure it starts with `https://www.linkedin.com/` or `https://linkedin.com/`

Comment (sambhavgupta0705) [reply extensions/linkedin/0.1.0/schema.json:2]: yes it will work

Comment (derberg) [reply extensions/linkedin/0.1.0/schema.json:2]: you used a wrong field

https://json-schema.org/understanding-json-schema/reference/regular_expressions

**Review by derberg — APPROVED** [review]
🚀

**Comment by derberg** [comment]
Anyone else wanna have a look?

With this PR @sambhavgupta0705 basically validates that https://github.com/asyncapi/spec-json-schemas?tab=readme-ov-file#extensions instruction works. Also simplistic contribution flow to https://github.com/asyncapi/extensions-catalog/pull/144 works as well.

Once we merge this I think we can say we have super minor nice MVP of extension catalog:
- we can onboard new extensions in https://github.com/asyncapi/extensions-catalog
- and we can even onboard them in official json schema https://github.com/asyncapi/spec-json-schemas
- we also have automation to automatically render docs to new extensions in docs https://www.asyncapi.com/docs/reference/extensions/x
- we also figured the best way to have extensions support in react component https://github.com/asyncapi/asyncapi-react/issues/819 that is already in progress to be developed and contributed by folks from SAP. I asked them that in their contribution they could out of the box provide extension from `x-x` and `x-linkedin` extensions.

File: extensions/linkedin/0.1.0/schema.json
Comment (smoya) [inline extensions/linkedin/0.1.0/schema.json:3]: IMHO Doesn't matter if users write `http` or rather `https`. Both are valid. It is linkedin who should decide if redirect will happen or not.

```suggestion
    "pattern": "^http(s)?://(www\\.)?linkedin\\.com.*$",
```

**Review by smoya — CHANGES_REQUESTED** [review]
Simple change requested here https://github.com/asyncapi/spec-json-schemas/pull/528#discussion_r1577551366

The rest LGTM 👍 🚀

**Review by smoya — APPROVED** [review]
LGTM! 🚀🌔

**Pull request merged**
